mci/app/app.py

```diff
@@ -5,38 +5,32 @@
 """
 
 import json
+
+from brighthive_authlib import OAuth2ProviderError
 from flask import Flask
+from flask_migrate import Migrate
 from flask_restful import Api
 from flask_sqlalchemy import SQLAlchemy
-from flask_migrate import Migrate
-from brighthive_authlib import OAuth2ProviderError
-from mci.config import ConfigurationFactory
 
-app = Flask(__name__)
-app.config.from_object(ConfigurationFactory.from_env())
-db = SQLAlchemy(app)
-migrate = Migrate(app, db)
-api = Api(app)
+from mci.api import (AddressResource, DispositionResource,
+                     EducationLevelResource, EmploymentStatusResource,
+                     EthnicityRaceResource, GenderResource,
+                     HealthCheckResource, SourceResource, UserDetailResource,
+                     UserResource)
+from mci.config import ConfigurationFactory
+from mci_database.db import db
 
-if db:
-    from mci.api import UserResource, UserDetailResource, HealthCheckResource, SourceResource,\
-        GenderResource, AddressResource, DispositionResource, EthnicityRaceResource,\
-        EmploymentStatusResource, EducationLevelResource
+def create_app():
```

Comment:
This pattern is great however you can wrap your `flask_restful` classes with a decorator to clean this up. Take a look at how I implement it with a seemingly circular imports.

https://github.com/brighthive/jdx-api/blob/master/jdxapi/app.py
https://github.com/brighthive/jdx-api/blob/master/jdxapi/routes/__init__.py
https://github.com/brighthive/jdx-api/blob/master/jdxapi/routes/framework_selections.py